org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/authentication/FormBasedAccessController.java

```diff
@@ -87,13 +91,11 @@ public void destroy() {
    */
   @SuppressWarnings({"squid:RedundantThrowsDeclarationCheck", "DuplicatedCode"}) // required so that overriding subclasses can throw ServletExceptions
   protected boolean handleAuthRequest(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
```

Comment:
A unit test covering several cases for handleAuthRequest() would be nice ;-)
E.g. using mocked request and expectation on response?

Reply:
Fixed in c23066ab4cc592f8bef8ff6d062c5ab7fd48d8a0